doc/protocol.md

```diff
@@ -571,6 +571,20 @@ network connection. The server may need these for statistics purposes. The value
 must be a nonnegative integer strictly less than 2^63.
 
 
+### JSON_ERROR
+
+    head  =  'json_error' <error_code> <message size> <session ident>
+    body  =  <message>
+
+Introduced in protocol version 4.
```

Comment:
This is not technically correct. Should we bump the protocol version for this new message?

Reply:
We are bumping the protocol version, so I'll update this.